### DISCUSSION TITLE
Cant create hobby account

### DISCUSSION THREAD

**Discussion by TwizztedOne** [vercel/community#3680] [Account Issue] [answered]
### Question

This user account is blocked.

Starting project
https://github.com/TwizztedOne/Evedencer.git

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
